### PR TITLE
initial spec for building libzabbix RPM out of openshift-tools

### PR DESCRIPTION
**Pull request by joelddiaz**

### CODE REVIEW

**Comment by joelddiaz** [comment]
@twiest @kwoodson let me know what you think of this (specifically the naming)

File: ansible/openshift-tools-ansible.spec [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# %commit is intended to be set by tito custom builders provided
+# in the .tito/lib directory. The values in this spec file will not be kept up to date.
+%{!?commit:
+%global commit c64d09e528ca433832c6b6e6f5c7734a9cc8ee6f
+}
```
Comment (twiest) [inline ansible/openshift-tools-ansible.spec]: Does this actually work?

Do we need to set something in tito for it to work?

**Comment by twiest** [comment]
@joelddiaz naming is fine by me. Rest looks good. Just had the one comment / question.

**Comment by joelddiaz** [comment]
@twiest: deleted the %{!?commit... stuff out of the spec. tito built the RPM fine without it.

**Comment by kwoodson** [comment]
:+1:

**Pull request merged**